src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMemberRow.tsx

```diff
@@ -10,59 +10,65 @@ import LoadingIndicator from 'app/components/loadingIndicator';
 import {PanelItem} from 'app/components/panels';
 import {IconCheckmark, IconClose, IconFlag, IconMail, IconSubtract} from 'app/icons';
 import {t, tct} from 'app/locale';
-import SentryTypes from 'app/sentryTypes';
 import space from 'app/styles/space';
+import {AvatarUser, Member} from 'app/types';
 import recreateRoute from 'app/utils/recreateRoute';
 
-export default class OrganizationMemberRow extends React.PureComponent {
-  static propTypes = {
-    routes: PropTypes.array,
-    // XXX: Spreading this does not work :(
-    member: SentryTypes.Member,
-    onRemove: PropTypes.func.isRequired,
-    onLeave: PropTypes.func.isRequired,
-    onSendInvite: PropTypes.func.isRequired,
-    orgName: PropTypes.string.isRequired,
-    memberCanLeave: PropTypes.bool,
-    requireLink: PropTypes.bool,
-    canRemoveMembers: PropTypes.bool,
-    canAddMembers: PropTypes.bool,
-    currentUser: SentryTypes.User,
-    status: PropTypes.oneOf(['', 'loading', 'success', 'error']),
-  };
+type Props = {
+  params: Record<string, string>;
+  routes: PlainRoute[];
```

Comment:
Should you intersect with `RouteComponentProps` to get `params` and `routes`?

Reply:
I actually had, but it injects a bit too much unfortuantely.